qt/scientific_interfaces/Inelastic/QENSFitting/FitDataPresenter.cpp

```diff
@@ -98,8 +98,14 @@ WorkspaceID FitDataPresenter::getNumberOfWorkspaces() const { return m_model->ge
 
 size_t FitDataPresenter::getNumberOfDomains() const { return m_model->getNumberOfDomains(); }
 
-FunctionModelSpectra FitDataPresenter::getSpectra(WorkspaceID workspaceID) const {
-  return m_model->getSpectra(workspaceID);
+QList<FunctionModelDataset> FitDataPresenter::getDatasets() const {
+  QList<FunctionModelDataset> datasets;
```

Comment:
Does not have to be in this PR. But `FunctionModelDataset` class could be refactored with  std strings.